packages/auth/src/Domain/Permission/Permission.ts

```diff
@@ -1,5 +1,12 @@
-export enum Permission {
+import { Uuid } from '../Uuid/Uuid'
+
+export enum PermissionName {
   SyncItems = 'SYNC_ITEMS',
```

Comment:
could we change the permission names to the ones that are used? 
https://github.com/standardnotes/auth/blob/develop/migrations/1624434102642-update_roles_and_permissions.ts#L17-L46

Reply:
Done!